Specs/DateTime/English/DateTimeModel.json

```diff
@@ -18684,36 +18685,22 @@
     "Context": {
       "ReferenceDateTime": "2016-11-07T00:00:00"
     },
+    "NotSupported": "javascript, python, java",
     "Results": [
       {
-        "Text": "every day",
+        "Text": "every day at 7:13 p.m.",
         "Start": 11,
-        "End": 19,
+        "End": 32,
         "TypeName": "datetimeV2.set",
         "Resolution": {
           "values": [
             {
-              "timex": "P1D",
+              "timex": "T19:13",
```

Comment:
Hmmm. Shouldn't the new timex include both terms?

Reply:
I'm ok with it as the recurrence is kind of implicit. "every time it is 19:13".
But I worry about other cases like "every workday at 19:13".

Reply:
Earlier version of set parser doesn't support cases like "every day at 5pm" as single entity, there were two different entities recognized with timex as set P1D and T19:30. So in default mode I have kept only T19:30 as timex for set resolution and for TASKSMODE timex type is "P1DT19:30".